shell/src/main/java/alluxio/cli/fsadmin/journal/QuorumInfoCommand.java

```diff
@@ -69,16 +70,21 @@ public int run(CommandLine cl) throws IOException {
     }
 
     GetQuorumInfoPResponse quorumInfo = jmClient.getQuorumInfo();
+    List<String[]> table = quorumInfo.getServerInfoList().stream().map(info -> new String[]{
```

Comment:
I would prefer `IsLeader` out of this table.
Instead, I believe printing the leader separately above this table is much cleaner.

Reply:
Transitioned printing the leader address in the header.
I still communicate the `isLeader` boolean through the RPC, but I do not print it to the command line. I instead use the boolean to get the leader's address and print said address as part of the header.